### PR TITLE
fix VkSparseMemoryBind offset check

### PR DESCRIPTION
**Pull request by ziga-lunarg**

### CODE REVIEW

File: layers/core_validation.cpp [hunk 1]
```diff
@@ -12514,16 +12514,21 @@ bool CoreChecks::PreCallValidateQueueBindSparse(VkQueue queue, uint32_t bindInfo
...
+                const VkSparseMemoryBind &memory_bind = image_opaque_bind.pBinds[sparse_bind_idx];
                 const auto *mem_info = Get<DEVICE_MEMORY_STATE>(memory_bind.memory);
                 if (mem_info) {
                     if (memory_bind.memoryOffset >= mem_info->alloc_info.allocationSize) {
-                        skip |= LogError(
-                            image_bind.image, "VUID-VkSparseMemoryBind-memoryOffset-01101",
-                            "vkQueueBindSparse(): pBindInfo[%u].pImageBinds[%u]: memoryOffset is not less than the size of memory",
-                            bind_idx, image_idx);
+                        skip |=
+                            LogError(image_opaque_bind.image, "VUID-VkSparseMemoryBind-memoryOffset-01101",
```
Comment (ncesario-lunarg) [inline layers/core_validation.cpp:12527]: Looking at the VUID more closely, I'm a bit confused now. I think this looks right, but shouldn't this be checked and produced for each bind type: `pBufferBinds`, `pImageOpaqueBinds`, `pImageBinds`?

**Review by ncesario-lunarg — COMMENTED** [review]
I think this looks right, but looks like maybe some cases are missing.

File: layers/core_validation.cpp (hunk 1 above)
Comment (ziga-lunarg) [reply layers/core_validation.cpp:12527]: I am unsure, the VUID is VUID-VkSparseMemoryBind-memoryOffset-01101 and has only VkSparseMemoryBind in it, so I don't know if this applies to VkSparseImageMemoryBind .

Comment (ncesario-lunarg) [reply layers/core_validation.cpp:12527]: Right, so looks like this should at least apply to `pBufferBinds` and `pImageOpaqueBinds`. I'm assuming this _should_ also apply to `pImageBinds`, but don't see an appropriate VUID for it. Maybe 01105 would work, but it's not quite the same.

**Review by ncesario-lunarg — APPROVED** [review]

**Pull request merged**